Description: On Debian systems the examples are in another directory.
 Thus modify description where to find more examples.
 Note: not to be forwarded upstream
Origin: vendor
Forwarded: not-needed
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2020-08-31

--- a/lib/HTML/Parser.pm
+++ b/lib/HTML/Parser.pm
@@ -1065,9 +1065,10 @@
   $p->parse_file(shift || die) || die $!;
   print "\n";
 
-More examples are found in the F<eg/> directory of the C<HTML-Parser>
-distribution: the program C<hrefsub> shows how you can edit all links
-found in a document; the program C<htextsub> shows how to edit the text only; the
+On a Debian box, more examples can be found in the
+/usr/share/doc/libhtml-parser-perl/examples directory.
+The program C<hrefsub> shows how you can edit all links
+found in a document and C<htextsub> how to edit the text only; the
 program C<hstrip> shows how you can strip out certain tags/elements
 and/or attributes; and the program C<htext> show how to obtain the
 plain text, but not any script/style content.
